bindings/java/src/main/java/org/openucx/jucx/NativeLibs.java

```diff
@@ -67,8 +67,9 @@ private static void loadLibrary(String resourceName) {
                 errorMessage = "Native code library failed to load: " + file.getName()
                     + ". " + ex.getLocalizedMessage();
             }
-
-            file.deleteOnExit();
+            if (url.getProtocol().equals("jar")) {
```

Comment:
@petro-rudenko if the protocol is jar, it means extractResource(url) is not doing anything?

Reply:
So it loads so from classpath. `extractResource(url)` will just open so lib. So we don't need to delete it: https://github.com/openucx/ucx/blob/master/bindings/java/src/main/java/org/openucx/jucx/NativeLibs.java#L84

Reply:
can we move file.deleteOnExit() inside extractResource?

Reply:
Ok it's already there: https://github.com/openucx/ucx/blob/master/bindings/java/src/main/java/org/openucx/jucx/NativeLibs.java#L102

No need that line at all.